.lagoon.yml

```diff
@@ -178,6 +178,15 @@ environments:
           - "go.cms-playground.dpl-cms.dplplat01.dpl.reload.dk":
               annotations:
                 nginx.ingress.kubernetes.io/proxy-buffer-size: 32k
+  pr-2679:
+    routes:
+      - varnish:
+          - "core-update.dpl-cms.dplplat01.dpl.reload.dk":
```

Comment:
"core-update" is a chosen name for this PR - it can be any name, but it **cannot** be pr-XXXX as lagoon will fail.